tests/fixtures/fixture_client.py

```diff
@@ -11,15 +11,6 @@
 api_token = os.getenv('ZENATON_API_TOKEN')
 app_env = os.getenv('ZENATON_APP_ENV')
 
-if not app_id:
-    raise Exception('Please include your ZENATON_APP_ID in the .env file')
```

Comment:
Instead of requiring an env, I'm starting with a client that cannot connect the agent nor the API and it's enough for most unit tests. For the ones that actually need this, I added a `skipif`.